caikit/runtime/service_generation/signature_parsing/parsers.py

```diff
@@ -30,12 +30,13 @@
 
 # Constants ##################################
 KNOWN_ARG_TYPES = {
-    "syntax_doc": "SyntaxPrediction",
     "producer_id": "ProducerId",
-    "raw_document": "RawDocument",
 }
 
-KNOWN_OUTPUT_TYPES = {}
+KNOWN_OUTPUT_TYPES = {
+    "SampleTaskPredict": "SampleOutputType",
+    "OtherTaskPredict": "OtherOutputType",
+}
```

Comment:
Let's make the call on keeping this at all, and if so it seems like a good time to make it configurable so that we don't leak test fixture names into the source code